.github/buildomat/jobs/tuf-repo.sh

```diff
@@ -174,8 +182,8 @@ path = "$rot_image_a"
 kind = "file"
 path = "$rot_image_b"
 [[artifact.${tufaceous_board}_sp]]
-name = "${tufaceous_board}_sp"
-version = "$sp_version"
+name = "$sp_caboose_board"
```

Comment:
I think this is fine but just want to note that the SP `BORD` values don't actually include the string "sp" - they're `gimlet-c`, `gimlet-d`, etc. The RoT `BORD` value is more clear (`oxide-rot-1`).

Reply:
Yeah, I think that's fine, since the artifact kind makes it clear that this is an SP image. We need a key that the control plane can reliably use to determine which SP image is correct for the hardware it's trying to update, and making it match what it can read from what's already installed is I think the most reliable we can get right now.